src/sentry/users/services/lost_password_hash/model.py

```diff
@@ -13,7 +16,7 @@ class RpcLostPasswordHash(RpcModel):
     id: int = -1
     user_id: int = -1
     hash: str = ""
-    date_added = datetime.datetime
+    date_added: datetime = Field(default_factory=timezone.now)
```

Comment:
I'm not super sure we want this to be the default. I could also remove this entirely and just assume the caller needs to provide one, but I this was failing tests for whatever reason 🤔

Reply:
I think it's a reasonable default.  Forcing the caller to provide one is reasonable in tests, but not in production, due to the reality that there will be (increasing) version drift between callers and receivers.

Reply:
Whatever default does it exist, it should be safe for the purpose of backwards compatibility, which depends on the use case.

Reply:
This default seems fine to me. Looking at the existing usage, we only create `RpcLostPasswordHash` by serializing an ORM object, which would have `date_added` set as well.